### DISCUSSION TITLE
Mirrors (bitbucket und gitlab) seit Juni nicht mehr aktuell

### DISCUSSION THREAD

**Discussion by station7** [Freetz-NG/freetz-ng#819] [Allgemein] [answered]
Leider sind die freetz-ng-repository-mirrors bei [bitbucket ](https://bitbucket.org/Freetz-NG/freetz-ng/commits/) und [gitlab](https://gitlab.com/Freetz-NG/freetz-ng/-/commits/master?ref_type=heads) nicht mehr aktuell. Würde mich sehr freuen, falls das jemand beheben könnte.

Vielen Dank!

**Comment by fda77** [comment, marked as answer]
bitbucket hatte wohl probleme und hat den hostkey getauscht
![image](https://github.com/Freetz-NG/freetz-ng/assets/471850/c4c3aaa0-a839-43a3-9fbc-abcbf12c8048)

**Comment by fda77** [comment]
und gitlab hatte einfach keine lust mehr:
![image](https://github.com/Freetz-NG/freetz-ng/assets/471850/01746ec1-c606-4912-a597-d4649e6db5fb)
Nach dem drücken auf den button gings nun wieder, ka ob das so bleibt

**Comment by station7** [comment]
Hallo cuma, 

perfekt, vielen Dank für den Fix!
